src/neuro_relay/linker.py

```
@@ -116,25 +154,33 @@ async def _handle_traffic(self):
-                            }
+                                "silent": True,  # Keep events quieter in production
+                            },
                         }
-                    
+
                     await self.nakurity_client.send_command_data(
-                        __import__("json").dumps(payload).encode()
+                        __import__("json").dumps(payload).encode(),
                     )
                 await asyncio.sleep(0)  # yield control
             except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)